components/Chip/index.vue

```diff
@@ -42,12 +41,16 @@ const chipSize = computed(() => {
     {{ text }}
     <button
       class="pb-1 pl-2 align-middle"
-      @click.prevent="emit('removeChip', text)"
+      @click.prevent="
```

Comment:
```suggestion
      @click.stop="
```
Would using [`v-on.stop`](https://vuejs.org/api/built-in-directives.html#v-on) instead of calling `stopPropagation()` not work?